### PR TITLE
Add exact terminal steps

### PR DESCRIPTION
**Pull request by jsncmgs1**
tendies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,21 +5,31 @@ This scans every ticker on the market, gets their last 5 months of volume histor
...
--get my script from GitHub
+Download my script:
+```
+$ git clone https://github.com/SamPom100/UnusualVolumeDetector.git
+```
 
--open the script in your IDE and install all required dependancies by typing pip install -r requirements.txt into the IDE's terminal. You can get to the the terminal on VSC by pressing CMD and ` at the same time.
+Get packages
+```
+$ pip install -r requirements.txt
```
Comment (luisott) [inline README.md]: pip3 here too?

Comment (jsncmgs1) [reply README.md]: I actually don't know python but that makes sense to me.

**Review by luisott — COMMENTED** [review]
I'm also just learning it now lol but I had to run pip3 if not it complained that the versions listed there didn't exist (I think it was defaulting to 2 and those versions maybe only apply to 3 🤷‍♂️ ?)

**Review by luisott — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (elizabethshirley) [reply README.md]: Pip is fine for new and fresh installations of python, since Python 2 has been sunset.

**Pull request closed without merging**